definitions/npm/redux-act_v1.x.x/flow_v0.34.x-/redux-act_v1.x.x.js

```diff
@@ -0,0 +1,49 @@
+declare module 'redux-act' {
+  declare module.exports: {
+    createReducer<ReducerState>(
+      handlers: {
+        [key: string]: (ReducerState, payload: mixed) => ReducerState,
+      },
+      defaultState?: ReducerState,
+    ): {
+      (): (state: ReducerState, payload?: mixed, meta?: mixed) => ReducerState,
+
+      // TODO: Document parameters & add tests
+      options: () => mixed,
+      has: (actionCreator: mixed) => boolean,
+      on: (actionCreator: mixed, reduceFunction: () => mixed) => void,
+      off: (actionCreator: mixed) => void,
```

Comment:
on and off return 'reducer'
https://github.com/pauldijou/redux-act/blob/master/src/createReducer.js#L32